compliance/components/ManualProcedures/component.yaml

```diff
@@ -44,32 +44,24 @@ satisfies:
   implementation_status: complete
   narrative:
     - text: |
-        Because the infrastructure is all code, Disaster Recovery into another region
-        should be simple, only requiring a change to the terraform code to change the
-        region and a restore of a database backup from the old region.
+        Because the infrastructure is all code, Disaster Recovery
+        should be simple, only requiring a relaunch and a restore of a database backup
+        from the old system.
 - control_key: PL-8
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: partial
   narrative:
     - text: |
-        The [DevSecOps Architecture](https://github.com/18F/gcp-appengine-template/blob/master/DEVSECOPS.md)
-        describes the overall system and security architecture.  In particular, the
-        [DevSecOps Requirements](https://github.com/18F/gcp-appengine-template/blob/master/DEVSECOPS.md#devsecops-requirements)
-        section goes into details about the DevSecOps security requirements that this project
-        fulfills.
-
-        XXX Be sure to add information about your app's security architecture either here or in
-        the [DevSecOps Architecture](https://github.com/18F/gcp-appengine-template/blob/master/DEVSECOPS.md)
-        document.
+        XXX Be sure to add information about your app's security architecture.
 - control_key: SA-22
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: complete
   narrative:
     - text: |
         XXX CircleCI requires you to specify a few specific component versions,
-        such as terraform, the Cloud SDK, any environments you use to do builds/deploys.
+        such as any environments you use to do builds/deploys.
         None are currently old, but you might want to create and document a periodic
```

Comment:
This line feeds more like a note rather than a suggested implementation of this control. Also is it CircleCIl? Or more like requirements.txt?

Reply:
It is a note.  All of these things that have XXX in there are sort of placeholders for the vendor to fill out as they write the application.  These files are not meant to actually get an ATO for the prototype, but to provide a good example of the structure of ATO documentation that they can use as they write their application.

Reply:
I'm not seeing anywhere that we make not that `XXX` is a note - can we replace the `XXX` with something that will help those taking a look for the first time understand that?

Reply:
Dang.  I thought I had that documented somewhere.  Thanks for pointing that out!